FrontEnd/css/ProfilePage.css

```diff
@@ -0,0 +1,125 @@
+body{
+    margin: 0;
+    padding: 0;
+    background-color: #370617;
```

Comment:
Consider making these color scheme colors variables at the top of the css page and then simply referencing them throughout the document. Would reduce redundancy and make it easier to change the color scheme in the future if we want to